### PR TITLE
Update Polish translation

### PR DESCRIPTION
**Pull request by knuxify**

### CODE REVIEW

File: pl_PL/LC_MESSAGES/trophies.po [hunk 1]
```diff
@@ -3,123 +3,123 @@
...
 msgid "TROPHY_DESC_weekly_topic_winner"
-msgstr "Submit a winning entry to Sudomemo's Weekly Topic"
+msgstr "Twój Flipnote wygrał Temat Tygodnia"
 
 msgid "TROPHY_NAME_folder"
 msgstr "Folder"
 
 msgid "TROPHY_DESC_folder"
-msgstr "Donate spare compute time to Folding@Home to help fight disease"
+msgstr "Wesprzyj nasz zespół na Folding@Home by pomóc w walce z chorobami"
```
Comment (knuxify) [inline pl_PL/LC_MESSAGES/trophies.po:119]: When translated back, this says "Support our team on Folding@Home". Is this a good way to put this?

Comment (sudofox) [reply pl_PL/LC_MESSAGES/trophies.po:119]: The focus should be purely on donating your computer's power and fighting disease.

**Comment by sudofox** [comment]
Hi! I'll review this soon :)

**Comment by sudofox** [comment]
Looking good, thanks! But...what's your Flipnote Studio ID?

**Comment by knuxify** [comment]
> Looking good, thanks! But...what's your Flipnote Studio ID?

9B552C708A06974C

**Pull request merged**